SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestReplicationConditions/TestReplicationConditionsActor.cpp

```diff
@@ -206,6 +209,12 @@ ATestReplicationConditionsActor_Physics::ATestReplicationConditionsActor_Physics
 	SetRootComponent(StaticComponent);
 }
 
+void ATestReplicationConditionsActor_Physics::OnRep_ReplicatedMovement()
+{
+	InitFakePhysics();
```

Comment:
As far as I could tell, I couldn't replicate any of this "fake physics" stuff that I create. I also got an error if I tried to create it within the constructor, and creating it in BeginPlay was too late. So my silly fix right now is to do it before replicating, and create it if it hasn't been yet.

Reply:
This will only get called on non-auth workers, do they need to set this up?